src/Core/Termination/CallGraph.idr

```diff
@@ -200,9 +255,10 @@ mutual
   mkChange : Defs -> Name ->
              (pats : List (Term vars)) ->
              (arg : Term vars) ->
-             List SizeChange
+             Core (List SizeChange)
```

Comment:
Minor thing but might be nice to take the fuel as an input here just to avoid the need to introduce the Core side effecty type for such a small thing.

Reply:
The `fuel` comes from `defs` argument on line 260 below.  It's now been a little while since I wrote this, but it looks like I wired in `Core` for the `lookupCtxtExact` in `sizeChange` (line 169 above). We need it to expand metas on the right hand side.

Reply:
Ah, gotcha. I missed the upstream Core addition for sizeChange. Thanks.